api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java

```diff
@@ -56,18 +59,51 @@ public DashboardRemoteServiceImpl(
         this.componentRepository = componentRepository;
     }
 
+    /**
+     * Creates a list of unique owners from the owner and owners requests
+     * @param request
+     * @return List<Owner> list of owners to be added to the dashboard
+     * @throws HygieiaException
+     */
+    private List<Owner> getOwners(DashboardRemoteRequest request) throws HygieiaException {
+        DashboardRemoteRequest.DashboardMetaData metaData = request.getMetaData();
+        Owner owner = metaData.getOwner();
+        List<Owner> owners = metaData.getOwners();
+
+        if (owner == null && owners == null) {
+            throw new HygieiaException("There are no owner/owners field in the request", HygieiaException.INVALID_CONFIGURATION);
+        }
+
+        if (owners == null) {
+            owners = new ArrayList<Owner>();
+            owners.add(owner);
+        } else if (owner != null) {
+            owners.add(owner);
+        }
+
+        Set<Owner> uniqueOwners = new HashSet<Owner>(owners);
+        return new ArrayList<Owner>(uniqueOwners);
+    }
+
     @Override
     public Dashboard remoteCreate(DashboardRemoteRequest request, boolean isUpdate) throws HygieiaException {
         Dashboard dashboard;
         Map<String, Widget> existingWidgets = new HashMap<>();
 
-        if (!userInfoService.isUserValid(request.getMetaData().getOwner().getUsername(), request.getMetaData().getOwner().getAuthType())) {
-            throw new HygieiaException("Invalid owner information or authentication type. Owner first needs to sign up in Hygieia", HygieiaException.BAD_DATA);
+        List<Owner> owners = getOwners(request);
+
+        for (Owner owner : owners) {
+            if (!userInfoService.isUserValid(owner.getUsername(), owner.getAuthType())) {
```

Comment:
Do we want to fail even if one of the owners passed in is bad?

Reply:
I would assume no, but that's up to us to decide, should I change it to instead remove the owner from the list?

Reply:
just skip if one of the owner is bad and proceed to the next owner passed in the list.